src/main/java/seedu/duke/command/AddNoteCommand.java

```diff
@@ -39,23 +42,43 @@ public AddNoteCommand(Note note) {
 
     @Override
     public String execute() {
+        String content = note.getContent();
+        boolean isInputSuccess = false;
+
         // Search for duplicates
         ArrayList<Note> filteredTaskList = (ArrayList<Note>) notebook.getNotes().stream()
-                .filter((s) -> s.getTitle().equals(note.getTitle()))
+                .filter((s) -> s.getTitle().equalsIgnoreCase(note.getTitle()))
                 .collect(toList());
 
         if (!filteredTaskList.isEmpty()) {
-            return COMMAND_UNSUCCESSFUL_MESSAGE;
+            return Formatter.formatString(COMMAND_UNSUCCESSFUL_MESSAGE, false);
+        }
+
+        // Get Content
+        if (content.isBlank()) {
+            do {
+                System.out.println("Enter Note:");
+                try {
+                    content = inputContent();
+                    isInputSuccess = true;
+                } catch (StringIndexOutOfBoundsException exception) {
+                    System.out.println(SystemException.ExceptionType.EXCEPTION_INVALID_END_INPUT);
+                }
+            } while (!isInputSuccess);
         }
 
+        // Edit the note
+        note.setContent(content);
+
         // Rebind the tags if there are duplicated tags
         tagManager.rebindTags(note);
         notebook.addNote(note);
 
         if (note.getTagsName().isBlank()) {
-            return COMMAND_SUCCESSFUL_MESSAGE + note.getTitle();
+            return Formatter.formatNote(COMMAND_SUCCESSFUL_MESSAGE + note.getTitle(), note);
         } else {
-            return COMMAND_SUCCESSFUL_MESSAGE + note.getTitle() + " " + note.getTagsName();
+            return Formatter.formatNote(COMMAND_SUCCESSFUL_MESSAGE + note.getTitle() + " "
+                    + note.getTagsName(), note);
```

Comment:
No need to check if getTagsName is blank. Will do the fix on the formatter side.

Reply:
will do it once ive merge this in. have yet to pull the new one. might have conflicts